tests/testthat/test_base_TuneWrapper.R

```diff
@@ -36,8 +36,9 @@ test_that("TuneWrapper", {
   lrn1 = makeLearner("classif.ksvm", predict.type = "prob")
   lrn2 = makeTuneWrapper(lrn1, resampling = makeResampleDesc("Holdout"), par.set = ps1, control = makeTuneControlGrid())
   expect_equal(lrn2$predict.type, "prob")
-  r = resample(lrn2, binaryclass.task, makeResampleDesc("Holdout"), measures = mlr::auc)
+  r = resample(lrn2, binaryclass.task, makeResampleDesc("CV"), measures = mlr::auc)
```

Comment:
Try to keep the unit tests as inexpensive as possible. Better use Holdout or CV with iters = 2.